src/System/include/BipedalLocomotion/System/VariablesHandler.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file VariablesHandler.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <string>
+#include <unordered_map>
+
+#include <iDynTree/Core/Utils.h>
+
+#ifndef BIPEDAL_LOCOMOTION_SYSTEM_ADVANCEABLE_VARIABLES_HANDLER_H
+#define BIPEDAL_LOCOMOTION_SYSTEM_ADVANCEABLE_VARIABLES_HANDLER_H
```

Comment:
I guess it should be ``BIPEDAL_LOCOMOTION_SYSTEM_VARIABLES_HANDLER_H``

Reply:
Yes thanks!

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/50/commits/63c1bc69a695bff77932407568f0cd6914f75e24